_posts/en/newsletters/2024-01-03-newsletter.md

```diff
@@ -0,0 +1,402 @@
+  a time when other users are unresponsive (whether unintentionally or
+  deliberately).  The typical way to construct such protocols is by
+  giving each user an offchain transaction that the user can broadcast
+  in case they want to exit.  That means, even in the best case, if five
+  users want to exit, they each need to broadcast a separate
+  transaction, and each of those transactions will have at least one
+  input and one output---a total of five inputs and five outputs.
+  Ingala suggests a way those users could effect a single exit with a
+  single transaction that could have a single input and five outputs,
+  giving them the typical [payment batching][topic payment batching]
+  reduction in transaction size by about 50%.
+
+  In complex multiparty contracts with very large numbers of users, the
+  reduction in onchain size can easily be significantly larger than 50%.
+  Even better, if the five active users simply wanted to move their
+  funds to a new shared UTXO involving just them, they could use a
+  single-input and single-output transaction, saving about 80% in the
+  case of five users or about 99% in the case of a hundred users.  That
+  huge savings for large groups of users moving their funds from one
+  contract to another may be critical when transaction feerates are high
+  and many users have relatively small balances in the contract. For
+  example 100 users each have a balance of 10,000 sats ($4 USD at the
+  time of writing); if they each individually had to pay transaction
+  fees to exit the contract and enter a new contract, then even with an
+  improbably small spending transaction size of 100 vbytes, a
+  transaction fee of 100 sats/vbyte would consume their entire balance.
+  If they are able to move their combined funds of 1 million sats in a
+  single 200 vbyte transaction at 100 sats/vbyte, then each individual
+  user will only pay 200 sats (2% of their balance).
+
+    The payment batching is accomplished by having one of the
+    participants in the multiparty contract protocol construct a spend
+    of the shared funds to the outputs agreed upon by the other active
+    participants.  The contract allows this---but only if the
+    constructing party funds a bond that they will forfeit if anyone can
+    prove that they spent the contract protocol's funds incorrectly; the
+    bond amount should be considerably more than the constructing party
+    can gain from attempting an incorrect transfer of funds.  If no one
+    generates a fraud proof showing the constructing party acted
+    inappropriately within a period of time, the bond is refunded to the
+    constructing party.  Ingala roughly describes how this feature could
+    be added to a multiparty contract protocol using [OP_CAT][],
+    `OP_CHECKCONTRACTVERIFY`, and amount introspection from the proposed
+    [MATT][] soft fork, with him noting that it would be easier with the
```

Comment:
```suggestion
    [MATT][] soft fork, and noted that it would be easier with the
```

Reply:
Leaving as-is.  There's a conjunctive "and" a few words earlier also following a comma and I don't want this clause to look like a list item.